cmds/bmp/create_baremetals_command_test.go

```diff
@@ -1,29 +1,36 @@
 package bmp_test
 
 import (
+	"errors"
+	"os"
+	"path/filepath"
+
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 
+	fakes "github.com/cloudfoundry-community/bosh-softlayer-tools/clients/fakes"
 	cmds "github.com/cloudfoundry-community/bosh-softlayer-tools/cmds"
 	bmp "github.com/cloudfoundry-community/bosh-softlayer-tools/cmds/bmp"
-
-	fakes "github.com/cloudfoundry-community/bosh-softlayer-tools/clients/fakes"
 )
 
 var _ = Describe("create-baremetals command", func() {
 
 	var (
-		args    []string
-		options cmds.Options
-		cmd     cmds.Command
-
+		args          []string
+		options       cmds.Options
+		cmd           cmds.Command
+		deployment    string
 		fakeBmpClient *fakes.FakeBmpClient
 	)
 
 	BeforeEach(func() {
 		args = []string{"bmp", "create-baremetals"}
+		wd, _ := os.Getwd()
+		deployment = filepath.Join(wd, "../..", "test_fixtures/bmp", "deployment.yml")
```

Comment:
So this should be written as:

```
deployment = filepath.Join(wd, "..",  "..", "test_fixtures", "bmp", "deployment.yml")
```
The point of `filepath.Join(...)` is to add the separator. I'll accept but please add a little chore to fix these next chance you get. Thx.